### PR TITLE
FIX - make HttpClient a single instance

### PR DESCRIPTION
**Pull request by newmediadev**
Use this technique to stop the app creating a new connection pool each time the about window is opened, as per best practice for HttpClient.

For more information on why you should not use using with HttpClient please read https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/

Also converted the Version tuple to a single model class.

### CODE REVIEW

File: Rapr/UpdateManager.cs [hunk 1]
```diff
@@ -2,36 +2,63 @@
...
+                .Add(new MediaTypeWithQualityHeaderValue("application/vnd.github.v3+json"));
+
+            httpClient.DefaultRequestHeaders
+                .Add("User-Agent", "System.Net.Http Agent");
+
+            var response = await httpClient.GetAsync(new Uri(versionInfoUrl)).ConfigureAwait(false);
+
+            if (response.IsSuccessStatusCode)
+            {
+                var releaseInfo = JObject.Parse(response.Content.ReadAsStringAsync().GetAwaiter().GetResult());
```
Comment (lostindark) [inline Rapr/UpdateManager.cs]: Do not call .GetAwaiter().GetResult() inside async method.

File: Rapr/AboutBox.cs [hunk 2]
```diff
@@ -53,13 +58,19 @@ public AboutBox()
             }
         }
 
+        private async Task<VersionInfo> getLatestVersion()
```
Comment (lostindark) [inline Rapr/AboutBox.cs]: Method should be pascal case.

File: Rapr/DSEForm.cs [hunk 3]
```diff
@@ -27,6 +27,8 @@ public partial class DSEForm : Form
         private Color savedForeColor;
         private OperationContext context = new OperationContext();
 
+        private static readonly IUpdateManager _updateManager = new UpdateManager();
```
Comment (lostindark) [inline Rapr/DSEForm.cs]: Why prefix with _?

File: Rapr/AboutBox.cs (hunk 2 above)
Comment (newmediadev) [reply Rapr/AboutBox.cs]: Addressed this

File: Rapr/DSEForm.cs (hunk 3 above)
Comment (newmediadev) [reply Rapr/DSEForm.cs]: It's a Resharper thing, I've used it for so long I'm in the habit of using it even when Resharper is off.

File: Rapr/UpdateManager.cs (hunk 1 above)
Comment (newmediadev) [reply Rapr/UpdateManager.cs]: Addressed.###

**Pull request merged**